vite.config.js

```diff
@@ -38,8 +39,12 @@ const overriddenComponentsToNodeModules = Object.fromEntries(
 
 // https://vitejs.dev/config/
 export default defineConfig({
+  define: {
+    "process.env": {},
+  },
```

Comment:
What is this for ?

Reply:
Ohh, good catch, probably some earlier version of the YAML parser or something. No longer needed, thanks!